docs/developer/releasing.md

```diff
@@ -113,25 +124,55 @@ For patch releases, the release shepard will:
 
 3. Create the patch release tag.
 
-After the release shepard pushes a new tag, they must [publish the release](#publishing-the-release).
+4. [Publish the release](#publishing-the-release)
 
 ### Updating code
 
-The codebase must be updated whenever a new release is being made to reference
-the upcoming release tag:
+The codebase must be updated whenever a new release is being made to reference the upcoming release tag.
 
-* Modify `CHANGELOG.md` with a new version number and its release date.
-* Go through the entire repository and find references to the previous release
-  version, updating them to reference the new version.
-* Run `make generate-manifests` and `make generate-dashboards` to update
-  manifests in case they are stale.
+#### Update the changelog
 
 NOTE: Any time CHANGELOG.md is updated for a release, it should first be done
```

Comment:
We should move this warning and re-word it a little. Any change done for a release should be done in main first and then moved to the release branch.

Reply:
Good point! I'll move it to the start of the "Updating code" section, like this:

```
### Updating code

The codebase must be updated whenever a new release is being made to reference the upcoming release tag.

NOTE: Any change done for a release should be done in `main` first and then moved to the release branch.
```